spec/api-app-spec.js

```diff
@@ -536,7 +536,7 @@ describe('app module', () => {
     ]
 
     before(function () {
-      if (process.platform === 'linux' || process.mas) this.skip()
+      if (process.platform !== 'win32') this.skip()
```

Comment:
this is surprising, shouldn't these tests run on mac?

Reply:
@nornagon the SMLogin APIs are the ones that failed on MAS circle like 75% of the time; now those will also be present on non-MAS builds so we don't want to run them

Reply:
Wait, so the _new_ APIs are the failing ones? I'd like to find a way that we can reliably test these APIs.

Reply:
@nornagon yes, unfortunately :(